src/test/java/GuiAppTest/NumberTextFieldTest.java

```diff
@@ -12,6 +12,6 @@ public void returnsSomeTextAsString() {
 
         String text = textField.getText();
 
-        assertTrue(text.equals("some text"));
+        assertTrue(text.equals("44"));
```

Comment:
This test is a bit confusing because of the random number 44!
I also noticed that you're just testing the test double with it. You're not actually instantiating any production code in this test.
Most of the time we don't need to test test doubles, especially if they're straight forward like the `TextFieldDouble`. There's no complicated logic in there really, so no need to write a test for it.